tests/functions_tests/test_arrays.py

```diff
@@ -37,6 +37,12 @@
      'args': {'pad_width': ((0, 0), (0, 0), (2, 2), (2, 2)),
               'mode': 'edge'},
      'name': 'pad_edge'},
+    {'ops': 'pad', 'input_shape': (1, 2, 3, 4),
+     'input_argname': 'x',
+     'args': {'pad_width': ((0, 0), (0, 0), (2, 2), (2, 2)),
```

Comment:
Not your fault, but could you try using a more varieties of `pad_width`? For example, it's unfortunate there is no imbalanced pads (e.g., `(1, 3)`)

Reply:
Added some test case.